app/models/solr_document.rb

```diff
@@ -25,7 +25,7 @@ class SolrDocument
     title: "title_ssm",
     author: "author_ssm",
     language: "language_ssim",
-    format: "format"
+    format: "format_ssi"
```

Comment:
I was actually going to junk the format_ssi field since it's not actually used. I think format is the right field?

Reply:
Ah yup, forgot to undo that. Thanks!

Reply:
Reverted back to "format".